### PR TITLE
add HeaderTopBarSearchCapi switch

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What does this change?
Add a switch for https://github.com/guardian/dotcom-rendering/pull/6629

### CODE REVIEW

File: common/app/experiments/Experiments.scala [hunk 1]
```diff
@@ -82,3 +83,12 @@ object BillboardsInMerchSlots
...
       participationGroup = Perc0E,
     )
+
+object HeaderTopBarSearchCapi
+    extends Experiment(
+      name = "header-top-bar-search-capi",
+      description = "Adds CAPI search to the top nav",
+      owners = Seq(Owner.withGithub("@guardian/dotcom-platform")),
+      sellByDate = LocalDate.of(2023, 2, 1),
+      participationGroup = Perc0E,
```
Comment (mxdvl) [inline common/app/experiments/Experiments.scala:93]: Opt-in only? There’s a switch as well, so it can always be turned off…

**Review by mxdvl — APPROVED** [review]
Found it!

File: common/app/experiments/Experiments.scala (hunk 1 above)
Comment (mxdvl) [reply common/app/experiments/Experiments.scala:93]: I guess if you want to be able to test something in PROD on your machine that’s the only way. But I’d consider being bold and rolling out to 1% straight away. Considering this button is very rarely clicked at all…

Comment (jamesgorrie) [reply common/app/experiments/Experiments.scala]: Thanks for reminding me that that is a useful way to think - I've updated this.

File: common/app/experiments/Experiments.scala [hunk 2]
```diff
@@ -82,3 +83,12 @@ object BillboardsInMerchSlots
...
       participationGroup = Perc0E,
     )
+
+object HeaderTopBarSearchCapi
+    extends Experiment(
+      name = "header-top-bar-search-capi",
+      description = "Adds CAPI search to the top nav",
+      owners = Seq(Owner.withGithub("@guardian/dotcom-platform")),
+      sellByDate = LocalDate.of(2023, 2, 1),
+      participationGroup = Perc1B,
```
Comment (mxdvl) [inline common/app/experiments/Experiments.scala:93]: Bold, I love it!

**Pull request merged**